docs/C2SMGuidelines.rst

```diff
@@ -47,116 +49,90 @@ The variety of different version-suffix for the cosmo-package:
+``c2sm-features`` branch are not yet provided but will be offered
+soon. We thus recommend using the ``spack devbuildcosmo`` command for
 now.
 
-So long story short:
-
-**Always use a valid git tag as a version-suffix when building
-software with** *spack install* **for production!**
-
-Spack dev-build (C2SM Guidelines)
----------------------------------
-
-In order to install software with *spack dev-build* one needs a
-local source code.  Spack will compile the code as it is locally
-present. Contrary to *spack install*, version-suffix (@master, @v2.7.9, etc,) does not have
-any effect on the code version compiled.
-To be safe always use *@mdev-build* and copy the executable after installation
-into the source-folder.
-
-So long story short:
-
-**Always store the local sources and the corresponding executable in
-the same location!**
-
-Installation
-^^^^^^^^^^^^
-
-Per default, Spack installs software under ``/$SCRATCH/spack-install``.
-On Piz Daint ``$SCRATCH`` undergoes regular cleanup with deletion of
-files older than 30 days. This may corrupt the internal Spack database
-and lead to unexpected behaviour of Spack.
-
-Change location for package installations
------------------------------------------
-
-Spack offers the possibility to overwrite the default installation
-directory. To do so create the file *~/.spack/config.yaml* and
-specify there an install directory that is not deleted regularly like
-*/project* or */store* in the following way:
+..  attention::
+    Always use a valid git tag as a version-suffix when building
+    software with ``spack install`` for production!
 
-.. code-block:: yaml
+Option 2: spack dev-build
+-------------------------
 
-  config:                                                                                                                     
-     install_tree: /project/s903/juckerj/spack-install/          
+In order to install software with ``spack dev-build``, one needs a
+local source code.  Spack will then compile the code as it is locally
+present. Contrary to ``spack install``, the version suffix
+(``@master``, ``@v2.7.9``, etc.) does not have any effect on the code version compiled.
+To be safe, always use ``dev-build`` and copy the executable after installation
+into the source folder.
 
-**Always change the installation directory to a location that is not
-wiped-out regularly!**
+..  attention::
+    Always store the local sources and the corresponding executable in
+    the same location!
 
 Running
 ^^^^^^^
 
 When used properly, Spack is able to manage many different
 configurations of a package along with the corresponding
-run-environment.
+run environment.
 
 Load run-environment of a package
 ---------------------------------
 
-Spack provides the command *spack load* to load the environment
+Spack provides the command ``spack load`` to load the environment
 needed to run a binary into your current shell. There are two
```

Comment:
It's not only useful for running a binary, think of python packages, which have no binary.
Also, I think we should put a link to https://spack.readthedocs.io/en/latest/command_index.html#spack-load here.

Reply:
Done